### DISCUSSION TITLE
Console exitcode

### DISCUSSION THREAD

**Discussion by lazna** [dannagle/PacketSender#381] [General]
regarding my investigation about https://github.com/dannagle/PacketSender/issues/81 just noticed, that PS produce exitcode (%errorlevel%) 4 in every case. is this an intend? Common behaviour of console programs is, that it produce exitcode 0 in case no problem durring runtime. And higher (documentated) number in case of problem.

**Comment by dannagle** [comment]
Bug confirmed on my ebd. Like you said, it should return zero unless an error occurred.

**Comment by dannagle** [comment]
created issue #382 for this.

**Comment by dannagle** [comment]
Fixed in the development branch. Will be fixed in the next release.
